stacks_queues/stack.py

```diff
@@ -21,18 +21,25 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        v = self.store.get_first()
+        self.store.remove_first()
+        return v
 
     def empty(self):
```

Comment:
✨ Alternatively, you could take advantage of the `LinkedList` class' `empty` method